rx-java/src/main/asciidoc/vertx-rx/java/index.adoc

```diff
@@ -89,6 +89,45 @@ an `rx.Observable` to a Vert.x read stream:
 {@link examples.NativeExamples#toReadStream}
 ----
 
+=== Write stream support
+
+A {@link io.vertx.core.streams.WriteStream}, like a `rx.Subscriber`, consumes items and collaborates with producers to avoid overflows.
```

Comment:
Perhaps say _to avoid unbounded pending write queue_ or something like this more specific.

Reply:
I went with:

```
A {@link io.vertx.core.streams.WriteStream}, like a `rx.Subscriber`, consumes items, and, when it can't keep-up, collaborates with the producer to avoid an ever-growing backlog.
```

I hope it's clear enough.